### PR TITLE
PLAT-548 OLTP consistency updates & add Getting Started

### PR DESCRIPTION
**Pull request by xenophenes**
Changes made:

- Consistency of intro settings, Configuration, & Extensions against the Standard Stack documentation
- Consistency update of introducing “Getting Started” section based on simple CRUD style workload.

### CODE REVIEW

**Comment by xenophenes** [comment]
cc @sjmiller609

File: src/content/docs/product/stacks/Transactional/oltp.md [hunk 1]
```diff
@@ -21,5 +21,183 @@ The following configurations automatically scale based on the size of cpu, memor
...
--   `pg_stat_statements` comes pre-installed and enabled.
+-   `pg_stat_statements` comes pre-installed and enabled. It provides statistics on SQL statements executed by the database, which helps users analyze query performance and identify areas for optimization.
+
 -   Extensions from [Trunk](https://pgt.dev) can be installed on-demand.
+
+## Getting started
+
+This guide will walk through setting up a transactional workload on Postgres using [Clickbench's "hits" dataset](https://github.com/ClickHouse/ClickBench?tab=readme-ov-file#history-and-motivation) (representing page views on a system) and the [Hydra](https://github.com/hydradatabase/hydra) Postgres extension.
+
+This guide will demonstrate the use of both extensions in optimizing the database for analytical queries.
```
Comment (ChuckHend) [inline src/content/docs/product/stacks/Transactional/oltp.md:32]: I see hydra referenced here, but then no where else in the guide. Should this section be part of the OLAP guide?

**Comment by ChuckHend** [comment]
Looks good @xenophenes! I walked through the entire example and all worked well for me. I left one note regarding Hydra/analytics that feels like it might be better fit for the OLAP guide.

**Comment by xenophenes** [comment]
Wow, thanks for catching that @ChuckHend , that section was actually somehow left over from the initial copy from the OLAP Getting Started guide (was copying the headers / structure of the guide for consistency). That's been fixed. Glad to hear the rest of the example worked well!

**Review by ChuckHend — APPROVED** [review]

**Comment by samay-sharma** [comment]
@ChuckHend : If this is still good to go, please merge.

**Pull request merged**